Makefile

```diff
@@ -41,6 +41,12 @@ fast_tests_diffusers:
 	python -m pip install .[tests]
 	python -m pytest tests/test_diffusers.py
 
+# Run unit and integration tests related to Image segmentation
+fast_tests_object_detection:
+	python -m pip install .[tests]
+	python -m pip install timm
```

Comment:
if no_timm is used for the test, is this needed?

Reply:
Yes, it is still needed. The model has called some API/modules from timm library

Reply:
then let's move this to here: https://github.com/huggingface/optimum-habana/blob/main/setup.py#L41-L51
doesn't need to be a separate pip install.